ci/test-app

```diff
@@ -12,35 +12,29 @@ cd ${HOME}/workspace/testgeomapfishapp/
 ./build
 docker-compose down
 docker-compose up -d
-docker-compose exec geoportal bash -c 'PGHOST=externaldb PGDATABASE=test wait-db;'
-docker-compose exec geoportal wait-db
-docker-compose exec geoportal alembic --config=alembic.ini --name=main upgrade head
-docker-compose exec geoportal alembic --config=alembic.ini --name=static upgrade head
-docker-compose exec geoportal create-demo-theme
-((docker-compose exec geoportal update-po en > \
-    geoportal/testgeomapfishapp_geoportal/locale/en/LC_MESSAGES/testgeomapfishapp_geoportal-client.po) \
-    || ci/run-dc-logs docker-compose exec geoportal update-po en)
-docker-compose exec geoportal update-po fr > \
-    geoportal/testgeomapfishapp_geoportal/locale/fr/LC_MESSAGES/testgeomapfishapp_geoportal-client.po
-docker-compose exec geoportal update-po de > \
-    geoportal/testgeomapfishapp_geoportal/locale/de/LC_MESSAGES/testgeomapfishapp_geoportal-client.po
-docker-compose exec geoportal theme2fts
+docker-compose exec -T geoportal bash -c 'PGHOST=externaldb PGDATABASE=test wait-db;'
+docker-compose exec -T geoportal wait-db
+docker-compose exec -T geoportal alembic --config=alembic.ini --name=main upgrade head
+docker-compose exec -T geoportal alembic --config=alembic.ini --name=static upgrade head
+docker-compose exec -T geoportal create-demo-theme
+docker-compose exec -T tools update-po $(id --user) $(id --group) en fr de
+docker-compose exec -T geoportal theme2fts
 
 # Commit the l10n files modifications
 # To prevent fail on modification files check
 #git diff
 #git add geoportal/testgeomapfishapp_geoportal/locale/*/LC_MESSAGES/testgeomapfishapp_geoportal-*.po
 #git commit -m "Upgrade the po files"
 
-ci/run-dc-logs docker-compose exec geoportal ci/waitwsgi https://front/themes
+ci/run-dc-logs docker-compose exec -T geoportal ci/waitwsgi https://front/themes
 for path in c2c/health_check c2c/health_check?max_level=9 c2c/health_check?checks=check_collector "layers/test/values/type enum" admin/layertree admin/layertree/children
 do
-    ci/run-dc-logs docker-compose exec geoportal ci/test-new-project https://front/${path}
-    #docker-compose exec geoportal curl --insecure https://front/c2c/debug/stacks?secret=c2c
+    ci/run-dc-logs docker-compose exec -T geoportal ci/test-new-project https://front/${path}
+    #docker-compose exec -T geoportal curl --insecure https://front/c2c/debug/stacks?secret=c2c
 done
-ci/run-dc-logs docker-compose exec geoportal ci/test-new-project 'http://qgisserver:8080/mapserv_proxy?SERVICE=WMS&REQUEST=GetCapabilities'
+ci/run-dc-logs docker-compose exec -T geoportal ci/test-new-project 'http://qgisserver:8080/mapserv_proxy?SERVICE=WMS&REQUEST=GetCapabilities'
```

Comment:
don't we need a WMS version in the request? Ok it is optional, but I don't know if we want to be sure if given versions are supported we could explicitely do a couple of tests.